src/Data/Nat/Primality.agda

```diff
@@ -8,80 +8,290 @@
 
 module Data.Nat.Primality where
 
-open import Data.Empty using (⊥)
+open import Data.Bool.Base using (Bool; true; false; not; T)
```

Comment:
Provisional; until the 'prelude' definitions of `{Non}Unit`, `NonTrivial` get moved to `Data.Nat.Base`...